main.go

```diff
@@ -250,6 +257,27 @@ func main() {
 	if err != nil {
 		setupLog.Error(err, "could not post Cluster Info to Hub")
 	}
+	sa := &corev1.ServiceAccount{
```

Comment:
Please move this code to the hub package and expose a func that can be called here.